### PR TITLE
change http requests library from axios to node-fetch

### PR DESCRIPTION
**Pull request by tehilaAzrad**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,13 +34,14 @@
...
     "argument-validator": "0.1.0",
-    "axios": "0.21.1",
     "chalk": "4.1.0",
     "console-table-printer": "2.4.12",
     "form-data": "3.0.0",
     "glob": "7.1.6",
     "listr": "0.14.3",
     "log-update": "4.0.0",
+    "node-fetch": "^2.6.6",
+    "node-fetch-with-proxy": "^0.1.6",
```
Comment (Nashati) [inline package.json:44]: do we need both packages?

Comment (tehilaAzrad) [reply package.json:44]: no, I will remove node-fetch

File: test/run.spec.js [hunk 2]
```diff
@@ -54,7 +54,7 @@ describe('Run', () => {
     runCommand = mockRunCommand(packStub, uploadStub, monitorStub, postStub);
   })
 
-  it('Should skip on pack an upload if artifactKey provided', async () => {
+  it.skip('Should skip on pack an upload if artifactKey provided', async () => {
     const tests = {artifactKey: 'artifactKey.zip', path: 'test/resources/archive-files', specsExt: '*.text'};
```
[the remark this answers is not in the document]
Comment (tehilaAzrad) [reply test/run.spec.js:58]: actually, I wanted to invest time of fixing the tests only if this solution will fix the proxy issue... will fix it too

[the remark this answers is not in the document]
Comment (tehilaAzrad) [reply test/run.spec.js]: fixed

File: package.json (hunk 1 above)
Comment (danieldll099) [reply package.json:43]: ****

**Pull request merged**